### PR TITLE
Docs updates.

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

**Comment by jeffswartz** [comment]
Note that the target of this PR is master. I assume that, for docs corrections, we do not need to bump the version in lib/opentok/version.rb, right?

**Comment by msach22** [comment]
@jeffswartz I think any change - even a trivial one in the readme should follow semantic versioning because documentation is equally important as adding functionality.

Everything on the master branch should be what's shipped with the release because it's what's available in the package.

**Comment by msach22** [comment]
@jeffswartz I'm pointing this to dev, can you please update this when you get a chance?

File: lib/opentok/sip.rb [hunk 1]
```diff
@@ -1,7 +1,42 @@
 require "opentok/client"
 
+# An object that lets you use the OpenTok SIP gateway.
 module OpenTok
   class Sip
+    # Dials a SIP gateway to input an audio-only stream into your OpenTok session.
+    # See the {https://tokbox.com/developer/guides/sip/ OpenTok SIP developer guide}.
+    #
+    # @example
+    #    opts = { "from" => "true",
```
Comment (msach22) [inline lib/opentok/sip.rb]: @jeffswartz should "from" be set to "true" or an example value?

**Pull request merged**